src/virtual-machine.js

```diff
@@ -1509,6 +1509,12 @@ class VirtualMachine extends EventEmitter {
             const variable = target.lookupVariableById(variableId);
             if (variable) {
                 variable.value = value;
+                // XXX: Temporary hack
```

Comment:
Not sure if this is necessary. Don't forget to check after all this variable API migration stuff is over.

Reply:
> Not sure if this is necessary. Don't forget to check after all this variable API migration stuff is over.

It is actually necessary, forget about it.